docs/accessibility.md

```diff
@@ -0,0 +1,94 @@
+# Accessibility
+
+**Microsoft MakeCode** includes some core features which promote greater accessibilty for everyone. The menus, the JavaScript editor and the documentation are accessible with the keyboard, assistive technologies, and a high contrast color mode. These improve the visibility of the UI elements and the navigation between them. The block editor is not accessible yet, so, accessible coding is done using just the [JavaScript editor](#javascript-editor).
+
```

Comment:
Is this the right way to link to the javascript-editor?

Reply:
it's linked at several places. I think it is.